### PR TITLE
[UI] Notifications Screen with mock data

### PR DESCRIPTION
**Pull request by AyaNady17**
## Description

Implemented notifications screen UI with mock data

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (does not change functionality, e.g. code style improvements, linting)
- [ ] Documentation update

## How Has This Been Tested?

tested manually 

Please include screenshots below if applicable.
Screenshot_1723498365

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

## Maintainer Checklist
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] Tag the PR with the appropriate labels

### CODE REVIEW

**Comment by AyaNady17** [comment]
please review @Aarush-Acharya  @vrundraval24

File: lib/views/new_screens/new_notifications_screen.dart [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+          ),
+          onPressed: () => Navigator.of(context).pop(),
+        ),
+        title: const Text('Notifications'),
+        actions: [
+          Padding(
+            padding: const EdgeInsets.only(right: 16.0, left: 16),
+            child: InkWell(
+              child: const CircleAvatar(
+                backgroundImage: NetworkImage(
```
Comment (chandansgowda) [inline lib/views/new_screens/new_notifications_screen.dart:31]: Let's use an asset image here

File: lib/views/new_screens/new_notifications_screen.dart [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+      return '${difference.inHours} hours ago';
+    } else if (difference.inDays < 7) {
+      return '${difference.inDays} days ago';
+    } else {
+      return dateTime.toIso8601String();
+    }
+  }
+}
+
+class NotificationModel {
```
Comment (chandansgowda) [inline lib/views/new_screens/new_notifications_screen.dart:127]: move this to models folder

**Review by chandansgowda — CHANGES_REQUESTED** [review]
minor changes

File: lib/views/new_screens/new_notifications_screen.dart (hunk 1 above)
Comment (AyaNady17) [reply lib/views/new_screens/new_notifications_screen.dart]: done

File: lib/views/new_screens/new_notifications_screen.dart (hunk 2 above)
Comment (AyaNady17) [reply lib/views/new_screens/new_notifications_screen.dart]: done

**Comment by Aarush-Acharya** [comment]
@chandansgowda  @AyaNady17  Looks good to me

**Comment by AyaNady17** [comment]
please check this @chandansgowda

**Pull request merged**